### PR TITLE
Update the readme with rules and cards

### LINKED ISSUE #1: Add Short Circuit card rules and NFC card mappings
@sjparkinson has this on the old Trello board.

Waiting

### PR DESCRIPTION
**Pull request by sjparkinson**
Resolves #1.

See https://github.com/mainthread-technology/short-circuit/blob/master/app/src/main/java/technology/mainthread/shortcircuit/activity/MainActivity.java#L201-L272 for the card mapping.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,58 @@
...
+The winner is the player with the most charge remaining.
+
+Tap cards on the battery that should be affected.
+
+## Card Mappings
+
+Card ID | Card Effect
+--------|------------
+1 | Charge 10%
+2 | Charge 10%
```
Comment (AndrewJack) [inline README.md:36]: Do 1 & 2 both drain 10%?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,58 @@
...
+3 | Charge 20%
+4 | Charge 30%
+5 | Drain 10%
+6 | Drain 10% * 2
```
Comment (AndrewJack) [inline README.md]: so 20% ?

File: README.md [hunk 3]
```diff
@@ -1,11 +1,58 @@
...
+7 | Drain 20%
+8 | Drain 30% * 2
```
Comment (AndrewJack) [inline README.md]: 60%?

File: README.md [hunk 4]
```diff
@@ -1,11 +1,58 @@
...
+9 | Drain 40%
+10 | Drain 50%
+11 | Drain half your current charge!
+12 | Steal 10% charge from another player
+13 | Steal 20% charge from another player
+14 | Steal 10% charge from every other player
+15 | Select another player to drain 20% charge
+16 | Select another player to drain 20% charge
```
Comment (AndrewJack) [inline README.md:50]: 15 & 16 are the same?

File: README.md [hunk 5]
```diff
@@ -1,11 +1,58 @@
...
+17 | Select another player to gain 10% charge
+18 | Select another player to drain 30% charge
+19 | Select another player to drain 30% charge
```
Comment (AndrewJack) [inline README.md:53]: same?

Comment (sjparkinson) [inline README.md:53]: Yep I believe they were, we had duplicates, kinda like love letter has too.

File: README.md (hunk 2 above)
Comment (sjparkinson) [inline README.md]: Not actually sure what `* 2` is here. Pulled it straight from the trello card.

**Comment by AndrewJack** [comment]
LGTM :+1: merge it

**Pull request merged**